components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java

```diff
@@ -80,10 +80,10 @@
     private  QueryManager metricsQueryManager;
     public StatusDashboardMetricsDBHandler() {
         Connection conn = null;
-        dataSource = DashboardDataHolder.getInstance().getMetricsDataSource();
+        dataSource = MonitoringDataHolder.getInstance().getMetricsDataSource();
         if (dataSource != null) {
             try {
-                conn= DashboardDataHolder.getInstance().getMetricsDataSource().getConnection();
+                conn= MonitoringDataHolder.getInstance().getMetricsDataSource().getConnection();
```

Comment:
@Yasara123 space before = is missing

Reply:
Fixed in 44e07c5322adee1cb3927ef2ae42d41caf1c3130